setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+
+description = """a library containing classes and functions so you could
```

Comment:
Usually this points to a file. Later on add something more elaborate :)

Reply:
yeah i know i should put the readme there but there isn't much in readme anyway...
when i bump the next version i'll work on the readme

Reply:
OK, good job!